csv/src/main/java/com/fasterxml/jackson/dataformat/csv/CsvParser.java

```diff
@@ -671,7 +675,7 @@ public boolean isExpectedNumberIntToken()
                     return true;
                 }
             } catch (IOException e) {
-                throw new RuntimeException(e);
+                throw new IllegalStateException(e);
```

Comment:
@cowtowncoder you've applied most of the changes that were in this PR - but this is the one extra bit: would it be possible to use IllegalStateException or UncheckedIOException here?

Reply:
Ah, yes, thank you for reminding me -- I think I'll go with `UncheckedIOException` as in another place.
I had vague recollection there was one more, this was it.